### DISCUSSION TITLE
ブループリントなどの埋め込みのサポート

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#297] [要望・アイデア・不具合]
優先度は高くない要望となりますが、ZennがFigmaなどと同時にUnrealEngineのブループリント埋め込みなどをサポートされたとアナウンスされていてそんなことできるのか・・・と結構びっくりしたのですが、もし可能でしたらその辺も埋め込みがサポートされたら記事を書くときに楽になったりするかもしれません・・・！

Zennのアップデート:  
https://twitter.com/zenn_dev/status/1569533116347465728?s=20&t=vPVhORcxLOVS5e45zCifsw

※以前UnrealEngineなどを入門でちょっと触っていたころには以下の記事みたくブループリント関係はスクショ取って貼っていたのですが、Twitterなどの埋め込み同様に埋め込めると執筆が楽になったり更新が楽になったり・・・するかもしれません・・・！

例 :
https://qiita.com/simonritchie/items/e8d84c2941b5bd2862f5#%E4%BD%8D%E7%BD%AE%E9%96%A2%E4%BF%82%E3%81%AE%E5%AE%9A%E6%95%B0%E3%81%A8%E5%A4%89%E6%95%B0%E3%82%92%E7%94%A8%E6%84%8F%E3%81%97%E3%81%A6%E3%81%8A%E3%81%8F

Twitterの引用リツイートなどを見ていても割と好意的な意見が多い気がいたします！

https://twitter.com/zenn_dev/status/1569533116347465728/retweets/with_comments

noteとかでも最近色々埋め込みできるようになっており、ブループリントもそうですが色々その辺拡充されると執筆が楽になっていいのかもと思っております・・・！

例 :
https://twitter.com/fladdict/status/1563323582390108160?s=20&t=P8kRpNuISVfX0uaAhwLu3w

少々その辺の埋め込みの仕組みやZennの仕様とかに詳しくないため、[Epic Games Japan](https://qiita.com/organizations/epicgamesjapan)の[Kaz Okadaさん](https://qiita.com/EGJ-Kaz_Okada)とかに連絡がTwitterなどで取れそうでしたらQiitaで埋め込みできるようになったら便利かどうかなどお聞きしてみてもよいかもしれません！  
（私はちょっとかじっている程度なので影響はそんなに無いのですが、UnrealEngineユーザーの皆さん達にとっては結構快適さの面で影響するかもしれません）

※ブループリントなどに関係した記事を書くのがZennの方が楽だ・・・となってUnrealEngine関係の記事をQiitaで今まで書かれている方々がZennに引っ越しされてしまう方が増えてしまってもQiitaの皆様にとって微妙でしょうし今までQiitaで書いてきた方々も引っ越しの手間などが面倒でしょうからもし埋め込みができるようになったら双方にとって良いのでは・・・と思ったため色々やることがあって大変かとは存じますが将来余裕ができた際などにご一考いただけますと幸いです🙇‍♂️

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
ご意見ありがとうございます🙏
埋め込み関連のリクエストは他にもいただいていおりますので、今回のご意見も合わせて検討させていただきます！🙇

**Comment by tsuzuki-takaaki** [comment]
@simon-ritchie
大変お待たせしました🙇‍♂️

bluepirntUEの埋め込みに対応しました！
https://qiita.com/release-notes/2025-07-18

**Reply by simon-ritchie** [reply]
@tsuzuki-takaaki 
おお…！ご対応ありがとうございます！！🙇‍♂️
